packages/playwright-core/src/client/browserType.ts

```diff
@@ -189,7 +188,7 @@ export class BrowserType extends ChannelOwner<channels.BrowserTypeChannel> imple
         return result.result;
       } else {
         closePipe();
-        throw new Error(`Timeout ${params.timeout}ms exceeded`);
+        throw new Error(`browserType.connect: Timeout ${params.timeout}ms exceeded`);
```

Comment:
Shouldn't the `wrapApiCall` add this prefix automatically?

Reply:
Hm, good question, let me investigate. I thought it was a pure client-side call site w/o wrap.